src/anbox/container/lxc_container.cpp

```diff
@@ -278,25 +307,28 @@ void LxcContainer::start(const Configuration &configuration) {
     rootfs_path = SystemConfiguration::instance().combined_rootfs_dir();
 
   DEBUG("Using rootfs path %s", rootfs_path);
-  set_config_item("lxc.rootfs.path", rootfs_path);
+  set_config_item(lxc_config_rootfs_path_key, rootfs_path);
 
-  set_config_item("lxc.log.level", "0");
+  set_config_item(lxc_config_log_level_key, "0");
   const auto log_path = SystemConfiguration::instance().log_dir();
-  set_config_item("lxc.log.file", utils::string_format("%s/container.log", log_path).c_str());
+  set_config_item(lxc_config_log_file_key, utils::string_format("%s/container.log", log_path).c_str());
+
+#ifndef ENABLE_LXC2_SUPPORT
+    // Dump the console output to disk to have a chance to debug early boot problems
+    set_config_item("lxc.console.logfile", utils::string_format("%s/console.log", log_path).c_str());
```

Comment:
lxc2 has `lxc.console.logfile`, but no `lxc.console.rotate`

Reply:
Log seems to fail on start up on lxc2 for some reason, even without rotate.

Reply:
@mariogrip `lxc.console.rotate` doesn't exist on LXC 2.x for sure. 

However I wouldn't recommend enabling `lxc.console.logfile` for LXC 2.x if not absolutely necessary as the implementation in LXC is the best and was improved to be based on a much better performing ring buffer in 3.x. Check https://discuss.linuxcontainers.org/t/lxc-3-0-0-has-been-released/1449